File: courses/csharp-introduction/game_machine.cs
// pro

using System;

public class Program
{
    public static void Main(string[] args)
    {
        int number = Convert.ToInt32(Console.ReadLine());

        Run(number);
    }

    // complete the method
    /**
        1. Shooter 🔫
        2. Puzzle 🧩
        3. Snake 🐍
    */
    static void Run(int num)
    {
        switch (num)
        {
            case 1:
                Console.WriteLine("Shooter");
                break;
            case 2:
                Console.WriteLine("Puzzle");
                break;
            case 3:
                Console.WriteLine("Snake");
                break;
        }
    }
}

File: courses/cpp-introduction/game_machine.cpp
// pro

#include <iostream>
using namespace std;

/**
 * 1. Shooter
 * 2. Puzzle
 * 3. Snake
*/

// complete the method
void run(int number) {
    switch (number) {
        case 1:
            cout << "Shooter" << endl;
            break;
        case 2:
            cout << "Puzzle" << endl;
            break;
        case 3:
            cout << "Snake" << endl;
            break;
    }
}

int main() {
    int number;
    cin >> number;
    run(number);
}
